docs/docs/testcase_report_datatests.md

```diff
@@ -0,0 +1,117 @@
+# Test Case Report
+
+## General Infromation
+Test Stage: ✅ Unit
+
+Test Date: Wed. Mar. 24, 2021
+
+Tester: Chelsea Hong, Cameron Ridderikhoff
+
+Test Case Description:  Testing incomplete, null, and broken data for the .earnings and .quarterly_earnings properties of ticker.py for the yfinance module.
+
+
+Test Case Number: 3
+
+Results: ✅ Passed
+
+## Introduction
+
+Requirement(s) to be tested: .earnings, .quarterly_earnings
+
+Roles and Responsibilities: Quality assurance team’s responsibility was to ensure quarterly and annual earnings were correctly calculated regardless of the state of the data. 
+
+
+
+Set Up Procedures:
+Retrieve codebase
+*Assume user has github account and git installed onto their machine*
+1. `git clone https://github.com/sazzy4o/yfinance.git`
+Setup Virtual Environment (recommended)
+3. Install venv for Python
+4. Create virtual environment
+`virtualenv <desired path>`
+4. Activate virtual environment (macOS)
+`source <desired path/bin/activate>`
+Install Necessary Modules
+5. `pip3 install unittest`
+6. `pip3 install . -editable`
+
+
+
+Stop Procedures:
+Running tests can be terminated by pressing:
+`CTRL+C`
+
+
+
+## Environmental Needs
+Hardware: N/A (any modern computer)
+
+Software: Python 3.4+, unittest module, yfinance module 
+
+
+
+Procedural Requirements:
+
+Ensure the “data” folder contains the correct json files. Must run `pip install . -editable` inside the yfinance folder in order to install yfinance’s required additional modules.
```

Comment:
If you add `cd` above then I don't think you need "Must run `pip install . -editable` inside the yfinance folder"